src/screens/ChatDetailPage/index.js

```diff
@@ -123,7 +124,6 @@ const ChatDetailPage = ({route}) => {
   }, []);
   React.useEffect(() => {
     searchUserMessages(channelClient.channel?.cid);
-    setParticipants(channelClient.channel?.state?.members, dispatch);
   }, [clients.client]);
   const searchUserMessages = async (channelID) => {
     const messages = await clients.client.search(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `setParticipants` has been removed in the new hunk. If this function is necessary for setting up participants in the chat, its removal might cause issues. Please ensure that its functionality is either no longer needed or has been replaced elsewhere in the code.
```diff
-    setParticipants(channelClient.channel?.state?.members, dispatch);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->